pool/session.go

```diff
@@ -86,11 +89,8 @@ func (p *Pool) withinContainerSession(
 		p.cache.Put(cacheKey, tok)
 	}
 
-	tok.ForVerb(verb)
-	tok.BindContainer(containerID)
-
 	// sign the token
-	if err := tok.Sign(signer); err != nil {
+	if err = tok.Sign(signer); err != nil {
```

Comment:
So it's signed for every request? I'd expect `cache` to have signed tokens, otherwise...